passes/techmap/abc9.cc

```diff
@@ -305,11 +297,23 @@ void abc9_module(RTLIL::Design *design, RTLIL::Module *module, std::string scrip
 	for (size_t pos = abc9_script.find("{W}"); pos != std::string::npos; pos = abc9_script.find("{W}", pos))
 		abc9_script = abc9_script.substr(0, pos) + wire_delay + abc9_script.substr(pos+3);
 
+	std::string C;
+	if (design->scratchpad.count("abc9.if.C"))
```

Comment:
Do you want to add the -C and -R options to the docstring?

Reply:
No, let's keep them as undocumented options for now... other scratchpad entries are not documented yet either.